addons/zeus/functions/fnc_moduleSimulation.sqf

```diff
@@ -0,0 +1,28 @@
+/*
+ * Author: Fisher
+ * Toggle Simulation on object.
+ *
+ * Arguments:
+ * 0: The module logic <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Public: No
+ */
+ 
+ #include "script_component.hpp"
+ 
+ params ["_logic"];
+ 
+ if !(local _logic) exitWith {};
```

Comment:
then make the command instead of `enableSimulationGlobal` just `enableSimulation` ?